<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_7" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QFrame" name="search_fr">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="search_input">
              <property name="minimumSize">
               <size>
                <width>491</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>491</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Nhập tên tệp</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="advanced_search">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../TEMP/test2203/Qss/icons/_icons1304.qrc">
                <normaloff>:/font_awesome_solid/icons/font_awesome/solid/sliders.png</normaloff>:/font_awesome_solid/icons/font_awesome/solid/sliders.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>15</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>Arrange</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>A - Z</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Z - A</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Date -&gt;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Date &lt;-</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="file_tab">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QTableWidget" name="tableWidget_2">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAsNeeded</enum>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>id</string>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>13</pointsize>
                 </font>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Tên</string>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>13</pointsize>
                 </font>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Loại</string>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>13</pointsize>
                 </font>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Ngày</string>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>13</pointsize>
                 </font>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Tag</string>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>13</pointsize>
                 </font>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Priority</string>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>13</pointsize>
                 </font>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Xóa</string>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>13</pointsize>
                 </font>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="add_btn">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">#add_btn
{
    background-color: #ffffff;
	border: 1px solid #6272a4;
	border-radius: 5px;	
    color: #000000;
}
</string>
               </property>
               <property name="text">
                <string>(+)</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCustomSlideMenu" name="info_right" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>0</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QPushButton" name="noti_x_lb_btn">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="noti_fr_btn">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="noti_btn">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../TEMP/test2203/Qss/icons/_icons1304.qrc">
             <normaloff>:/font_awesome_regular/icons/font_awesome/regular/bell.png</normaloff>:/font_awesome_regular/icons/font_awesome/regular/bell.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="note_txt">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="noti_save_btn">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="new_tag_wg" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="new_tag_fr">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="new_tag_cbb">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>15</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="new_tag_btn">
           <property name="maximumSize">
            <size>
             <width>23</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="info_fr">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>130</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>130</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_22">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="name_lb">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>44</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Tên</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_15">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="size_lb">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="dir_lb">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Đường dẫn</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomSlideMenu</class>
   <extends>QWidget</extends>
   <header location="global">Custom_Widgets.QCustomSlideMenu.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../TEMP/test2203/Qss/icons/_icons1304.qrc"/>
 </resources>
 <connections/>
</ui>
